src/apps/rDSN.monitor/app_package/static/js/analyzer.js

```diff
@@ -0,0 +1,420 @@
+    }
+};
+
+function Counter2List(machine, app, section, counter, index) {
+    $(".list-group.remove-list-box").append('<li class="list-group-item" id="' + machine.replace(':','_') + index + '"><span class="glyphicon glyphicon-remove" onclick="$(\'#' + machine.replace(':','_') + index + '\').remove();"></span>' + machine + ' * ' + app + ' * ' + section + ' * ' + counter+ '</li>');
+    counterSelected.push({machine: machine, name: machine + ' * ' + app + ' * ' + section + ' * ' + counter, index:index});
+};
+
+function List2List() {
+    function containsCounter(obj, list) {
+    var i;
+    for (i = 0; i < list.length; i++) {
+        if (list[i].name === obj.name && list[i].machine === obj.machine) {
+            return true;
+        }
+    }
+    return false;
+    }
+
+    $(".list-group.remove-list-box li").remove();
+    for (counter in counterSelected) {
+        var newCounter = {machine: counterSelected[counter].machine, name: counterSelected[counter].name, index: counterSelected[counter].index};
+        if (!containsCounter(newCounter,counterList))
+        {
+            counterList.push(newCounter);
+            //$("#counterListAll").append('<li class="list-group-item counter-main"><pre>' + counterSelected[counter].name + '<span class="glyphicon glyphicon-remove pull-right" aria-hidden="true" onclick="$(this).parent().parent().remove();counterList.splice(counterList.indexOf(' + '{machine:\'' + counterSelected[counter].machine + '\', name:\'' +  counterSelected[counter].name + '\', index:' + counterSelected[counter].index + '}),1)"></span> <span class="glyphicon glyphicon-duplicate pull-right" aria-hidden="true" onclick="if($(&quot;#newcountertext&quot;).length) {return;}$(this).parent().parent().after(\'<li class=&quot;list-group-item counter-main &quot; id=&quot;newcounterli&quot;><input type=&quot;text&quot; id=&quot;newcountertext&quot;></li>\');$(&quot;#newcountertext&quot;).val(&quot;' + counterSelected[counter].name + '&quot;);$(&quot;#newcountertext&quot;).attr(&quot;size&quot;,' + counterSelected[counter].name.length + ');duplicateCounter();"></span></pre></li>');
+            $("#counterListAll").append('<li class="list-group-item counter-main"><pre>' + counterSelected[counter].name + '<span class="glyphicon glyphicon-remove pull-right" aria-hidden="true" onclick="$(this).parent().parent().remove();counterList.splice(counterList.indexOf(' + '{machine:\'' + counterSelected[counter].machine + '\', name:\'' +  counterSelected[counter].name + '\', index:' + counterSelected[counter].index + '}),1)"></span> </pre></li>');
```

Comment:
there are still many mixed code of html and js objects. I understand it is not easy for a quick fix, but let's work hard towards the other way.